src/index.tsx

```diff
@@ -42,28 +42,30 @@ function Updaters() {
   )
 }
 
-ReactDOM.render(
-  <StrictMode>
-    <Provider store={store}>
-      <HashRouter>
-        <LanguageProvider>
-          <Web3Provider>
-            <Blocklist>
-              <BlockNumberProvider>
-                <Updaters />
-                <ThemeProvider>
-                  <ThemedGlobalStyle />
-                  <App />
-                </ThemeProvider>
-              </BlockNumberProvider>
-            </Blocklist>
-          </Web3Provider>
-        </LanguageProvider>
-      </HashRouter>
-    </Provider>
-  </StrictMode>,
-  document.getElementById('root')
-)
+const container = document.getElementById('root')
+
+if (container)
```

Comment:
If there is no container then this should fail (by throwing an error), which I'd expect React to do on it's own. I think this is the rare case where you should assert that container exists on line 45

    const container = document.getElementById('root')!

Reply:
done, just used type assertion since non-null postfix operator is disabled in lint checks (generates error).